### PR TITLE
Added markers option to run tests matching pytest markers expression.

### PR DESCRIPTION
**Pull request by narmaku**
Added rhel_only fixture and updated tests cases that are only applicable to RHEL images.
Marked test cases as "pub", which are only applicable to published cloud images.
Improved HTML report with new summary element to know which markers have been used.
Improved HTML report column "Test Case" so that it displays a shorter name of the test executed.

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    pub: Tests that have to be run on images that are already published into cloud providers' marketplaces.
```
Comment (jabia99) [inline pytest.ini:3]: Apostrophe typo

**Review by jabia99 — COMMENTED** [review]
Looks great! Does the fixture work as expected?

File: pytest.ini (hunk 1 above)
Comment (narmaku) [reply pytest.ini:3]: Ad discussed via Slack, this is also valid grammatically speaking.

**Comment by narmaku** [comment]
> Looks great! Does the fixture work as expected?

@jabia99 , yes, but remember this is not a fixture, but a marker :+1:

**Pull request merged**